src/display/display_utils.js

```diff
@@ -338,7 +338,7 @@ const LinkTarget = {
 
 /**
  * Adds various attributes (href, title, target, rel) to hyperlinks.
- * @param {HTMLLinkElement} link - The link element.
+ * @param {HTMLAnchorElement} link - The link element.
  * @param {ExternalLinkParameters} params
  */
 function addLinkAttributes(link, { url, target, rel, enabled = true } = {}) {
```

Comment:
Considering how this helper function is now being used, with the other changes in the PR, we might be able to simply move it into the `web/` folder instead. @timvandermeij What do you think?